rootfs/api/serializers.py

```diff
@@ -474,3 +475,18 @@ class Meta:
         """Metadata options for a :class:`AppSettingsSerializer`."""
         model = models.AppSettings
         fields = '__all__'
+
+    def validate_whitelist(self, data):
```

Comment:
Alright - I'm just trying to make sure we're not maintaining stuff we don't have to... If that package can't allow for a mixed use field than we have to do our own